IssueTracker/IssueTracker.FileSystem/Repository/BolbStorageProvider.cs

```diff
@@ -41,8 +41,11 @@ public async Task UploadFileAsync(Models.File file)
     {
         if (file == null)
             throw new ArgumentException("You don't have files!");
-
-        await ContainerClient.CreateIfNotExistsAsync();
+        if (file.BlobName == String.Empty || file.BlobName == null)
+            throw new ArgumentException("Invalid BlobName");
+        if (file.Content == null)
+            throw new ArgumentException("Invalid Content");
+        //await ContainerClient.CreateIfNotExistsAsync();
```

Comment:
keep the code clean

Reply:
I created a variable where the will be stocked the return.